photon_installer/isoInstaller.py

```diff
@@ -43,9 +43,9 @@ def __init__(self, options):
             if arg.startswith("ks="):
                 if not ks_path:
                     ks_path = arg[len("ks="):]
-            elif arg.startswith("repo="):
-                if not repo_path:
-                    repo_path = arg[len("repo="):]
```

Comment:
let's continue to support single "repo=" for existing users

Reply:
I think it's time we create stable-4, stable-5 branches in installer.
But the catch is how do we install both versions in same system? I think we should not break backward compatibility until we figure out a way.

Reply:
It would be valuable if installer can support multiple Photon versions at the same time.

We should maintain backward compatibility up to some extend. And deprecate only the features which are expensive to maintain. Deprecation should be also done in  gradient way. Initially throw an warning, later make it more attractive such as ask for confirmation of using deprecated option and removing it ultimately.